MAINTAINERS.md

```diff
@@ -0,0 +1,17 @@
+# Introduction
+
+This document provides a list of the maintainers for each area of development in the oneDPL project.
+
+| Feature               | Maintainer          | Github ID |
+| --------------------- | ------------------- | -------- |
+| C++ standard policies and CPU backends | Dan Hoeflinger | @danhoeflinger |
+| SYCL device policies and SYCL backends | Mikhail Dvorskiy<br>Adam Fidel | @MikeDvorskiy<br>@adamfidel | 
+| Tested Standard C++ APIs & Utility Function Object Classes | Sergey Kopienko | @SergeyKopeinko |
+| C++17 standard algorithms | Mikhail Dvorskiy<br>Adam Fidel<br>SergeyKopienko<br>Dmitriy Sobolev | @MikeDvorskiy<br>@adamfidel<br>@SergeyKopienko<br>@dmitriy-sobolev |
```

Comment:
Julian probably fits into this one as well?

Reply:
I updated that list.

Reply:
It looks like the ordering got misaligned between Sergey and Julian.

Reply:
Done. Thanks for the catch.